opentelemetry-instrumentation/pyproject.toml

```diff
@@ -25,8 +25,9 @@ classifiers = [
 ]
 dependencies = [
   "opentelemetry-api ~= 1.4",
-  "setuptools >= 16.0",
   "wrapt >= 1.0.0, < 2.0.0",
+  "importlib-metadata >= 6.0, < 7.0",
```

Comment:
the upper bound probably should be `< 8.0`, see https://github.com/open-telemetry/opentelemetry-python/issues/3570 (or maybe no boundary at all, but not sure about that)

Reply:
Although I don't like version constraints, in this case I would leave it because of cpython's reputation for changing its interface 🤣